index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Portfolio Website</title>
+</head>
+
+<body>
+  <div class="header">
+    <div class="placeHolder">
+      <h1 class="head">Welcome</h1>
+    </div>
+    <div class="nbtn1">
+
+      <div class="v1"></div>
+      <div class="v2"></div>
+      <div class="v3"></div>
+
+    </div>
+  </div>
+
+  <div class="container">
```

Comment:
- [ ] The container of the section should be a `section` tag. * It represents a generic standalone section of the document, so please use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.
- [ ] The section should have the proper background image as seen in the figma design (decorations are always background images, not `img` tags). * If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- [ ] Notice in the image below, you are not using the correct font for the headline text.
- [ ] Please remove the green border around the headline section container. Also, remove the `margin-left: 11%` on `.headline` class because it makes that section not position well in comparison to the figma design. 


Your work | Figma
--- | ---
![image](https://user-images.githubusercontent.com/84156714/219690815-8a6361d9-c188-49ce-bf0b-54b872515d50.png) | ![image](https://user-images.githubusercontent.com/84156714/219690943-20f98999-6f8a-4584-86c5-226f923a445e.png)

Notice the difference in background. You can export this background illustration from Figma.